frontend/apps/reader/modules/readerlink.lua

```diff
@@ -834,8 +834,8 @@ function ReaderLink:onGotoLink(link, neglect_current_location, allow_footnote_po
     end
     logger.dbg("ReaderLink:onGotoLink: External link:", link_url)
 
-    local scheme = link_url:match("^(%w+)://")
-    local is_handled_external_link = scheme and util.arrayContains(self.handledSchemes, scheme)
+    local scheme = link_url:match("^(%w[%w+%-.]*):")
+    local is_handled_external_link = scheme and util.arrayContains(self.handledSchemes, scheme:lower())
```

Comment:
I'm a bit too late to comment on the first PR, so doing it on the second :)
"handled" sounds to me like "done being processed".
"supported" would read less strange, like "we can process it, we'll do that later".

Reply:
I suppose I said something similar in https://github.com/koreader/koreader/pull/11889#discussion_r1614873484

But handled can make sense as well because passing it off to the system doesn't include much supporting for example. I haven't double checked what all the specific behaviors are right now though.

Reply:
Yes, "supported" is good. Done.